### ISSUE TITLE
Fixed point description in manual

### ISSUE THREAD

**Issue by DavidRoy** [NERC-CEH/dunes-app#68]
From Andy Nelson:

As many of my fixed point photography locations will be in publicly accessible locations, I want to use this aspect of CitSci to raise awareness of the project and encourage participation in it. Taking fixed point photographs is something that can easily be done by members of the public, using instructions in the app, without guidance and training. The manual states that “Fixed point photographs should be taken at least once a year, in April or May, at preselected locations.” Could this be amended to something along these lines… “Fixed point photographs should be taken at least once a year, in April or May, at preselected locations. Further photographs can be taken at these locations and uploaded at any time of year.”?

**Comment by LaurenceJonesES** [comment]
Yes, we can amend the manual to reflect this. This will be done in a subsequent iteration of the manual.
